pep-0684.rst

```diff
@@ -0,0 +1,640 @@
+PEP: 684
+Title: A Per-Interpreter GIL
+Author: Eric Snow <ericsnowcurrently@gmail.com>
+Discussions-To: python-dev@python.org
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 08-Mar-2022
+Python-Version: 3.11
+Post-History: 08-Mar-2022
```

Comment:
The relevant PR, #2358 , hasn't _quite_ been formally merged yet, but it should be by the time this is. Assuming it is, you'll want to update this to be a link to the thread you posted.

```suggestion
Post-History: `08-Mar-2022 <https://example.com/thread_1>`__
```

Reply:
What is `DIscussions-To` for then?

Reply:
Clearly and unambiguously linking to the _active_ canonical PEP discussion thread (which almost always is in practice the latest one posted, but doesn't necessarily _have_ to be). We _could_ (and might in the future) parse the `Post-History` field, extract the latest and inject the link into the `Discussions-To` field in the rendered PEP, but there are a few things we have to take care of first, namely merging the outstanding PRs related to documenting, conforming and validating the new format, finishing and deploying the PEP 676 implementation, and then discussing and implementing that functionality, so for now there's a modest bit of duplication here.